backend/src/chats/chats.controller.ts

```diff
@@ -25,9 +25,11 @@ export class ChatsController {
   }
 
   @Post()
-  create(@Body() dto: CreateChatDto, @User() user: UserEntity) {
-    dto.userIds.push(user.id);
-    return this.chatsService.create(dto);
+  async create(
+    @Body() createchatsDto: CreateChatDto,
+    @User() user: UserEntity,
+  ): Promise<ChatEntity> {
+    return this.chatsService.create(createchatsDto, user);
```

Comment:
legal que adicionou o `async`, mas acho que faltou o `await` 🤣 
```suggestion
    return await this.chatsService.create(createchatsDto, user);
```